server/src/uuid/model/entity/mod.rs

```diff
@@ -639,41 +662,46 @@ impl Entity {
             )
             .save(&mut transaction)
             .await?;
-        }
+        } else {
+            let taxonomy_term_id = payload
+                .input
+                .taxonomy_term_id
+                .ok_or(EntityCreateError::TaxonomyTermIdError)?;
 
-        let entity_revision = Entity::add_revision(
-            &entity_add_revision_mutation::Payload {
-                input: entity_add_revision_mutation::Input {
-                    changes: payload.input.changes.clone(),
-                    entity_id,
-                    needs_review: false,
-                    subscribe_this: payload.input.subscribe_this,
-                    subscribe_this_by_email: payload.input.subscribe_this_by_email,
-                    fields: payload.input.fields.clone(),
-                },
-                revision_type: EntityRevisionType::from(payload.entity_type.clone()),
-                user_id: payload.user_id,
-            },
-            &mut transaction,
-        )
-        .await
-        .unwrap();
+            let current_last_position = sqlx::query!(
+                r#"
+                    SELECT position FROM term_taxonomy_entity
+                    WHERE term_taxonomy_id = ?
+                    ORDER BY position DESC
+                    LIMIT 1
+                "#,
+                taxonomy_term_id
+            )
+            .fetch_one(&mut transaction)
+            .await?
+            .position as i32;
 
-        sqlx::query!(
-            r#"
-                UPDATE entity
-                    SET current_revision_id = ?
-                    WHERE id = ?
-            "#,
-            entity_revision.id,
-            entity_id
-        )
-        .execute(&mut transaction)
-        .await?;
+            sqlx::query!(
+                r#"
+                    INSERT INTO term_taxonomy_entity (entity_id, term_taxonomy_id, position)
+                    VALUES (?, ?, ?)
+                "#,
+                entity_id,
+                taxonomy_term_id,
+                current_last_position + 1
+            )
+            .execute(&mut transaction)
+            .await?;
 
-        CreateEntityEventPayload::new(entity_id, payload.user_id, instance_id)
+            CreateTaxonomyLinkEventPayload::new(
+                entity_id,
+                payload.input.taxonomy_term_id.unwrap(),
```

Comment:
```suggestion
                taxonomy_term_id,
```

Reply:
done in [bc2b8c1](https://github.com/serlo/serlo.org-database-layer/pull/181/commits/bc2b8c14b8bd1212269e5c245a807cac0e9c08af)